rotkehlchen/api/rest.py

```diff
@@ -449,12 +449,15 @@ def set_settings(self, settings: ModifiableDBSettings) -> Response:
 
         with self.rotkehlchen.data.db.conn.read_ctx() as cursor:
             new_settings = process_result(self.rotkehlchen.get_settings(cursor))
-        result_dict = {'result': new_settings, 'message': ''}
+            cache = process_result(self.rotkehlchen.get_cache(cursor))
+        result_dict = {'result': new_settings | cache, 'message': ''}
         return api_response(result=result_dict, status_code=HTTPStatus.OK)
 
     def get_settings(self) -> Response:
         with self.rotkehlchen.data.db.conn.read_ctx() as cursor:
-            result_dict = _wrap_in_ok_result(process_result(self.rotkehlchen.get_settings(cursor)))
+            settings = self.rotkehlchen.get_settings(cursor)
+            cache = self.rotkehlchen.get_cache(cursor)
```

Comment:
why do you need it? for the `last_balance_save`? if is only one key maybe you could create a method to query only certains keys exposed to the frontend and return them instead of everything